### PR TITLE
Update-hyperlinks

### PR DESCRIPTION
**Pull request by cthomasONRR**
[:sunglasses: PREVIEW](https://nrrd-preview.app.cloud.gov/sites/Update-hyperlinks/)

Changes proposed in this pull request:

**onrr.gov hyperlinks located at:**
- [x]  https://revenuedata.doi.gov/downloads
- [x]  https://revenuedata.doi.gov/how-revenue-works/federal-laws/
- [x]  https://revenuedata.doi.gov/how-revenue-works/native-american-production/
- [x]  https://revenuedata.doi.gov/how-revenue-works/native-american-revenue/

**External hyperlinks located at:**
- [x] https://revenuedata.doi.gov/how-revenue-works/federal-laws/
- [x] https://revenuedata.doi.gov/how-revenue-works/native-american-ownership-governance/
- [x] https://revenuedata.doi.gov/how-revenue-works/native-american-production/
- [x] https://revenuedata.doi.gov/how-revenue-works/offshore-oil-gas/
- [x] https://revenuedata.doi.gov/how-revenue-works/aml-reclamation-program
- [x] https://revenuedata.doi.gov/how-revenue-works/lwcf
- [x] https://revenuedata.doi.gov/how-revenue-works/audits-and-assurances
- [x] Footer

**PDF map hyperlink**
- [x] https://revenuedata.doi.govs/how-revenue-works/gomesa

### CODE REVIEW

**Comment by cthomasONRR** [comment]
@Maroyafaied I was able to take my time and check my work (50 links in total were updated). Everything looks good and I'm ok to merge this PR into dev.

**Comment by Maroyafaied** [comment]
@cthomasONRR  I reviewed several webpages, all looked good. I'll merge after I merge Jeff's branch to dev and update this branch with dev. Thanks

**Review by Maroyafaied — APPROVED** [review]

**Pull request merged**
